src/components/Header/constants/pages.ts

```diff
@@ -0,0 +1,10 @@
+import { RoutesPath } from '../../Router/routesPath';
+import { RoutesName, Page } from '../types/pages.type';
```

Comment:
use `import type` for importing types

Reply:
but in this case, we will have two files with the same name and it will be not convenient